client/my-sites/email/email-management/style.scss

```diff
@@ -472,3 +472,20 @@
 		}
 	}
 }
+
+.empty-content {
```

Comment:
This targets the entire component, which will have ramifications elsewhere in Calypso - presume it should be more specific?

Reply:
Ah sorry, it was my understanding these styles will only be applied when the email management page loads, i.e. `client/my-sites/email/email-management/style.scss` is only loaded in this case and these styles aren't applied elsewhere. I'll look to be more specific here though, thanks.

Reply:
Thanks for spotting this, @Aurorum 👍 This becomes problematic when users navigate to other pages. While these styles will only be loaded when users visit this page, they won't unload when they navigate elsewhere.